app.rb

```diff
@@ -0,0 +1,158 @@
+require_relative './book'
+require_relative './classroom'
+require_relative './decorator'
+require_relative './person'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def start
+    selection
+  end
+
+  # list all books
+  def list_books
+    puts 'Sorry, there are no books available'
```

Comment:
- [ ] This line will show up whenever we call this function. You might add a check there so that the message is displayed only if there are no books.

Reply:
Thank you for the review, I have implemented this 😃